manual_test/cases.md

```diff
@@ -104,3 +104,20 @@ __Pass:__
 - [ ] Win
 - [ ] OSX
 - [ ] Linux
+
+# 6. Font Rendering
+
+## 2.1 Validate editor.fontWeight
+
+Prerequisite:
+- In configuration file, key "editor.fontFamily" either set to a __non absolute__ font name or absent (Ideally test both).
```

Comment:
I stated _non absolute_ because I suspect that loading absolute files might bring to the surface some expected bugs.
Indeed, loading a specific font file instead of a font family is bit like a pandora box because then oni2  lacks the weight / italic / bold variants which gonna be needed elsewhere.
Actually, perhaps it would be a good thing remove the ability to load a specific font file ? 🤔

Reply:
Thanks for adding the case here, this is perfect!

> Indeed, loading a specific font file instead of a font family is bit like a pandora box because then oni2 lacks the weight / italic / bold variants which gonna be needed elsewhere.

Good point. Some files will have the variants; but most won't - and then we miss out on bold/italic.

> Actually, perhaps it would be a good thing remove the ability to load a specific font file ? 

This might be reasonable - would simplify things a bit, now that we support font families. I removed the '...absolute path' verbiage from the settings to deprecate this. I wish I had a good way to check if any users were using absolute file paths; but will keep this in mind.